Makefile

```diff
@@ -97,6 +114,15 @@ else
 	go build -mod=readonly $(CORAL_BUILD_FLAGS) -o build/coral ./cmd/wasmcli
 endif
 
+build-gaiaflex: go.sum
+ifeq ($(OS),Windows_NT)
+    # wasmd nodes not supported on windows, maybe the cli?
```

Comment:
Wrong indent here (must be tabs). Doesn't this break Makefiles?

Reply:
Fixed